CHANGELOG.md

```diff
@@ -0,0 +1,172 @@
+# Changelog
+
+This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [0.4.9]() - 2019-10-23
+
+### Changed
+
+- Only print out number of configs and rules when running with verbose flag
+- Match let and const to var in javascript:
+    ```
+    var $F = "hello"
+    ```
+    will now match any of the following expressions:
+    ```javascript
+    var foo = "hello";
+    let bar = "hello";
+    const baz = "hello";
+    ```
+
+### Added
+
+- Print out --dump-ast
+- Print out version with `--version`
+- Allow ... in arrays
+    ```
+    [..., 1]
+    ```
+    will now match
+    ```
+    [3, 2, 1]
+    ```
+- Support Metavariable match on keyword arguments in python:
+    ```
+    foo(..., $K=$B, ...)
+    ```
+    will now match
+    ```
+    foo(1, 2, bar=baz, 3)
+    ```
+- Support python fstring variable substitution
+- Constant propogation in javascript:
+    ```
+    api("literal");
+    ```
+    will now match with any of the following:
+    ```javascript
+    api("literal");
+
+    const LITERAL = "literal";
+    api(LITERAL);
+
+    const LIT = "lit";
+    api(LIT + "eral");
+
+    const LIT = "lit";
+    api(`${LIT}eral`);
+    ```
+
+- Deep statement matching
+- Unified import resolution in python:
+    ```
+    import foo.bar.baz
+    ```
+
+    will not match any of the following statements:
+
+    ```python
+    import foo.bar.baz
+    import foo.bar.baz.qux
+    import foo.bar.baz as flob
+    import foo.bar.baz.qux as flob
+    from foo.bar import baz
+    from foo.bar.baz import qux
+    from foo.bar import baz as flob
+    from foo.bar.bax import qux as flob
+    ```
+- Support lambda expressions in javascript #288
```

Comment:
I'm don't really understand the change here #288

Reply:
Support for anonymous JavaScript functions in patterns